spec.v1.yml

```diff
@@ -2,9 +2,29 @@ swagger: '2.0'
 info:
   version: '1.0'
   title: Sunnah.com API
+  description: |-
+    Sunnah.com API offers access to verified data of collections of hadith of the Prophet (SAWS). Here you will find a list of the available endpoints in our REST API.
+
+    You will need an API key to access this data, which you may request from https://sunnah.com/contact.
+  contact:
+    name: Sunnah.com
+    url: 'https://sunnah.com/contact'
+    email: sunnah@iman.net
```

Comment:
I would rather not include an email address if that's possible at all.

Reply:
Removed